content/en/user-guide/aws/docdb/index.md

```diff
@@ -0,0 +1,230 @@
+{{< /command >}}
+
+```yaml
+{
+  "DBCluster": {
+    "DBClusterIdentifier": "test-docdb-cluster",
+    "DBClusterParameterGroup": "default.docdb",
+    "Status": "available",
+    "Endpoint": "localhost:39045",
+    "MultiAZ": false,
+    "Engine": "docdb",
+    "Port": 39045,
+    "MasterUsername": "test",
+    "DBClusterMembers": [],
+    "VpcSecurityGroups": [
+      {
+        "VpcSecurityGroupId": "sg-a30edea1f7da6ff90",
+        "Status": "active"
+      }
+    ],
+    "StorageEncrypted": false,
+    "DBClusterArn": "arn:aws:rds:us-east-1:000000000000:cluster:test-docdb-cluster"
+  }
+}
+```
+
+If we break down the previous command, we can identify:
+
+- `docdb`: The command related to Amazon DocumentDB for the `AWS CLI`.
+- `create-db-cluster`: The command to create an Amazon DocumentDB cluster.
+- `--db-cluster-identifier test-docdb-cluster`: Specifies the unique identifier for the DocumentDB
+  cluster. In this case, it is set to `test-docdb-cluster`. You can customize this identifier to a
+  name of your choice.
+- `--engine docdb`: Specifies the database engine. Here, it is set to `docdb`, indicating the use of
+  Amazon DocumentDB.
+
+Notice in the `DBClusterMembers` field of the cluster description that there are no other databases
+created.
+To create a new database, we can use the `create-db-instance` command, like in this example:
+
+{{< command >}}
+$ awslocal docdb create-db-instance --db-instance-identifier test-company
+--db-instance-class db.r5.large --engine docdb --db-cluster-identifier test-docdb-cluster
+{{< /command >}}
+```yaml
+{
+  "DBInstance": {
+    "DBInstanceIdentifier": "test-docdb-instance",
+    "DBInstanceClass": "db.r5.large",
+    "Engine": "docdb",
+    "DBInstanceStatus": "creating",
+    "Endpoint": {
+      "Address": "localhost",
+      "Port": 50761
+    },
+    "InstanceCreateTime": "2022-10-28T04:27:35.917000+00:00",
+    "PreferredBackupWindow": "03:50-04:20",
+    "BackupRetentionPeriod": 1,
+    "VpcSecurityGroups": [ ,
+      "AvailabilityZone": "us-east-1a",
+      "PreferredMaintenanceWindow": "wed:06:38-wed:07:08",
+      "EngineVersion": "12.34",
+      "AutoMinorVersionUpgrade": false,
+      "PubliclyAccessible": false,
+      "StatusInfos": [],
+      "DBClusterIdentifier": "test-docdb-cluster",
+      "StorageEncrypted": false,
+      "DbiResourceId": "db-M5ENSHXFPU6XHZ4G4ZEI5QIO2U",
+      "CopyTagsToSnapshot": false,
+      "DBInstanceArn": "arn:aws:rds:us-east-1:000000000000:db:test-docdb-instance",
+      "EnabledCloudwatchLogsExports": []
+      }
+      }
+```
+
+Some noticeable fields:
+
+- `--db-instance-identifier test-company`: Represents the unique identifier of the newly created
+  database
+- `--db-instance-class db.r5.large`: Is the instance type or class for the Amazon DocumentDB
+  instance. It determines the compute
+  and memory capacity allocated to the instance. `db.r5.large` refers to a specific instance type in
```

Comment:
@HarshCasper help me out here, is this `db instance class` mocked or ignored? is it relevant to describe it in any way?

Reply:
the `db-instance-class` will be returned for the `describe-db-instance`, but it doesn't change the behavior. So I would argue it's both: mocked + ignored (because we don't change the actual deployment based on this setting). 

Personally, I think it's not relevant for the docs, as we currently do not have any parity checks for this parameter. On AWS certain configuration combinations are not allowed (especially when it comes to cluster + instance combinations).